core/src/env/srml/types.rs

```diff
@@ -31,14 +31,34 @@ use parity_codec::{
 
 /// The SRML fundamental types.
 #[allow(unused)]
+#[cfg_attr(feature = "test-env", derive(Debug, Clone, PartialEq, Eq))]
```

Comment:
Why only implement those trait when `test-env` feature is enabled? Please add rational.